### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by olawale-o**
## In this PR, we did the following
-- Create indexes on **visits** and **owners** table

**EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;**
  - BEFORE
first_query

  - AFTER
first_query_optimized

**EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;**
 - BEFORE
second_query

 - AFTER
second_query_optimized

**EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';**
  - BEFORE
third_query

  - AFTER
third_query_optimized

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -93,4 +93,14 @@ CREATE TABLE visits (
...
+
+-- Vet clinic database: database performance audit
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX idx_visits_animal_id ON visits (animal_id);
+CREATE INDEX idx_visits_vets_id ON visits (vets_id);
+
+CREATE INDEX idx_owners_email ON owners (email);
```
Comment (DeliceLydia) [inline schema.sql]: ## Optional

- [ ] I suggest making the index on the case insensitive version of the email, 
As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));`

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (olawale-o) [reply schema.sql]: Hi @DeliceLydia,
Wow!!!!. These will be very helpful.

Comment (olawale-o) [reply schema.sql]: done

**Comment by olawale-o** [comment]
Hi @DeliceLydia 
Thank you for your time and effort on the review

**Pull request merged**